examples/cp-all-in-one/docker-compose.yml

```diff
@@ -2,7 +2,7 @@
 version: '2'
 services:
   zookeeper:
-    image: confluentinc/cp-zookeeper:5.0.0-beta30
+    image: confluentinc/cp-zookeeper
```

Comment:
... running from latest is a worst practice. Why don't we just pin to the correct release tag?

Reply:
+1000 this is what I've been asking for in Slack.

Reply:
@aayars So technically, the code within cp-all-in-one should point to latest release.
However, being in master makes it a bit ambiguous  since master holds -SNAPSHOT version.

I will pin it to 5.0.0, automating via tooling is a WIP.